lib/parsers/ltx.js

```diff
@@ -223,6 +234,7 @@ inherits(SaxLtx, EventEmitter)
 
 SaxLtx.prototype.end = function (data) {
   if (data) {
+    this._setLast(true)
```

Comment:
```suggestion
    isLastChunk = true;
```

Reply:
Was my initial choice, but standard fails with "'isLastChunk' is not defined" if done like this (unless isLastChunk is defined outside the SaxLtx function, eg. under the states constants).